api/tests/integration/infrastructure/repositories/assessment-result-repository_test.js

```diff
@@ -155,4 +158,135 @@ describe('Integration | Repository | AssessmentResult', function() {
       expect(mostRecentAssessmentResult).to.be.null;
     });
   });
+
+  describe('#getAssessmentResultReadModel', () => {
+
+    describe('when the given certification course id is correct', () => {
+
+      it('should return assessment results', async () => {
+        // given
```

Comment:
'should return assessment results read model' ?

Reply:
Vu ensemble, du coup, on enlève le read-model d'AssessmentResult (on renvoie bien un assessment result)